packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/edge/MultiLabelEdge.tsx

```diff
@@ -78,8 +79,39 @@ export const MultiLabelEdge = memo(
       return null;
     }
 
-    const { x: sourceX, y: sourceY } = getHandleCoordinatesByPosition(sourceNode, sourcePosition, sourceHandleId ?? '');
-    const { x: targetX, y: targetY } = getHandleCoordinatesByPosition(targetNode, targetPosition, targetHandleId ?? '');
+    let { x: sourceX, y: sourceY } = getHandleCoordinatesByPosition(sourceNode, sourcePosition, sourceHandleId ?? '');
+    let { x: targetX, y: targetY } = getHandleCoordinatesByPosition(targetNode, targetPosition, targetHandleId ?? '');
+
+    const handleRadius = 2;
```

Comment:
Is there some reasons you did not put this in getHandleCoordinatesByPosition ?

Reply:
Yes. As told in private, changing `getHandleCoordinatesByPosition` fixes the edges's extremities (which we want), but also changes the coordinates of the handles themselves when the edge is selected (which we don't want).